packages/client-python/tests/test_engine.py

```
@@ -0,0 +1,791 @@
+            assert await db.get('nonexistent') is None
+
+    @pytest.mark.asyncio
+    async def test_find_running_returns_alive_instance(self, tmp_home):
+        async with StateDB() as db:
+            await db.register('live', os.getpid(), 5565, '3.1.0', 'user')
+            result = await db.find_running()
+            assert result is not None
+            assert result['id'] == 'live'
+
+    @pytest.mark.asyncio
+    async def test_find_running_cleans_stale(self, tmp_home):
```

Ruff (0.15.7)

[warning] Unused method argument: `tmp_home`

(ARG002)